druid/src/lib.rs

```diff
@@ -125,6 +125,7 @@ pub mod lens;
 mod localization;
 mod menu;
 mod mouse;
+#[cfg(not(target_arch = "wasm32"))]
 #[cfg(test)]
 mod tests;
```

Comment:
Is there a benefit for making the tests work on wasm?
On a quick glance this seems to test nothing platform specific @cmyr?
If it does tho, there should be an issue opened for it.

Reply:
I am not sure about this one. I left it out for the time being because it requires changes in `piet` to get this to compile, (I submitted a PR to add the missing types to piet in https://github.com/linebender/piet/pull/167) and I'm not really familiar with the testing code. I suspect it can be used for testing although it needs to be run in a special way for wasm and not through `cargo test`, but I'm not sure. Is this appropriate for a separate PR? For reference there is [some code](https://github.com/linebender/piet/tree/master/piet-web/examples/basic) testing the wasm backend in piet.

Reply:
Sure, this can be done in a later PR, if it makes sens to do so.
Which is what I hoped @cmyr would enlighten me about.

Reply:
yea, happy for this to be followup.